OTAcceleratorCore/OTOneToOneCommunicator.m

```diff
@@ -180,23 +182,31 @@ - (void)notifiyAllWithSignal:(OTCommunicationSignal)signal error:(NSError *)erro
 
 #pragma mark - OTSessionDelegate
 -(void)sessionDidConnect:(OTSession*)session {
-    
-    [[LoggingWrapper sharedInstance].logger setSessionId:session.sessionId
+    LoggingWrapper *loggingWrapper = [LoggingWrapper sharedInstance];
+    [loggingWrapper.logger setSessionId:session.sessionId
                                             connectionId:session.connection.connectionId
                                                partnerId:@([self.session.apiKey integerValue])];
+    [loggingWrapper.logger logEventAction:KLogActionConnect
+                                                 variation:KLogVariationSuccess
+                                                completion:nil];
```

Comment:
This is a duplicate log which is captured in `connect` method. That's why I only use `StartComm` and `StopComm`